src/cursor.rs

```diff
@@ -469,9 +469,12 @@ impl Cursor {
     /// # Return value
     ///
     /// Returns a vector containing the BSON documents that were read.
-    pub fn next_batch(&mut self) -> Result<Vec<bson::Document>> {
-        let n = self.batch_size;
-        self.next_n(n)
+    pub fn get_current_batch(&mut self) -> Result<Vec<bson::Document>> {
```

Comment:
It feels somewhat ambiguous whether the grouping of that name is "[drain or get] batch" or "[drain] or [get next batch]". I know that the latter is somewhat nonsensical in this context, but it still feels like an awkward name that requires some logical overhead. Maybe something like `drain_current_batch`?